src/Presentation/Nop.Web/Areas/Admin/Views/Order/List.cshtml

```diff
@@ -413,6 +413,10 @@
                                     return '<span class="grid-report-item ' + color + '">' + data + '</span >';
                                 }
 
+                                function renderColumnCustomer(data, type, row, meta) {
+                                    return `${row.CustomerFullName}<br /><a href="/Admin/Customer/Edit/${row.CustomerId}">${data}</a>`;
```

Comment:
@imatoria You should use `Url.Content()` to define a link instead of raw text. See example [here](https://github.com/nopSolutions/nopCommerce/blob/develop/src/Presentation/Nop.Web/Areas/Admin/Views/ShoppingCart/CurrentCarts.cshtml#L244)